fedmsg_meta_fedora_infrastructure/tests/buildsys.py

```diff
@@ -45,33 +45,34 @@
 #Task 9053697 is canceled
 #"""
 
-_build_long_form_fail = """Package:    64tass-1.51.727-1.fc22
-Status:     failed
-Built by:   sharkcz
-ID:         288888
-Started:    Sat, 14 Mar 2015 19:09:10 UTC
-Finished:   Sat, 14 Mar 2015 19:11:02 UTC
-
-Closed tasks:
--------------
-Task 1755171 on fedora3.s390.bos.redhat.com
-Task Type: build (noarch)
-Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755171
-
-error building package (arch s390), mock exited with status 1; see build.log for more information
-
-Task 1755172 on fedora3.s390.bos.redhat.com
-Task Type: buildArch (s390)
-Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755172
-
-error building package (arch s390), mock exited with status 1; see build.log for more information
-
-Task 1755173 on fedora2.s390.bos.redhat.com
-Task Type: buildArch (s390x)
-Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755173
-
-error building package (arch s390x), mock exited with status 1; see build.log for more information
-"""
+## Same here.  the state in koji changes which makes this test change often.
+#_build_long_form_fail = """Package:    64tass-1.51.727-1.fc22
+#Status:     failed
+#Built by:   sharkcz
+#ID:         288888
+#Started:    Sat, 14 Mar 2015 19:09:10 UTC
+#Finished:   Sat, 14 Mar 2015 19:11:02 UTC
+#
+#Closed tasks:
+#-------------
+#Task 1755171 on fedora3.s390.bos.redhat.com
+#Task Type: build (noarch)
+#Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755171
+#
+#error building package (arch s390), mock exited with status 1; see build.log for more information
+#
+#Task 1755172 on fedora3.s390.bos.redhat.com
+#Task Type: buildArch (s390)
+#Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755172
+#
+#error building package (arch s390), mock exited with status 1; see build.log for more information
+#
+#Task 1755173 on fedora2.s390.bos.redhat.com
+#Task Type: buildArch (s390x)
+#Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755173
+#
+#error building package (arch s390x), mock exited with status 1; see build.log for more information
+#"""
```

Comment:
It was.  That test is permanently failing now :(  See the comment at the top:  ``## Same here.  the state in koji changes which makes this test change often.``